aspnet/fundamentals/dependency-injection.rst

```diff
@@ -158,21 +147,20 @@ Observe which of the ``OperationId`` values varies within a request, and between
 
 - *Transient* objects are always different; a new instance is provided to every controller and every service.
 - *Scoped* objects are the same within a request, but different across different requests
-- *Singleton* objects are the same for every object and every request
-- *Instance* objects are the same for every object and every request, and are the exact instance that was specified in ``ConfigureServices``
+- *Singleton* objects are the same for every object and every request (regardless of whether an instance is provided in ``ConfigureServices``)
 
-Request Services and Application Services
------------------------------------------
+Request Services
+----------------
 
-The services available within an ASP.NET request from ``HttpContext`` fall into two collections: *ApplicationServices* and *RequestServices*.
+The services available within an ASP.NET request from ``HttpContext`` are exposed through the ``RequestServices`` collection.
 
-.. image:: dependency-injection/_static/application_request_services.png
+.. image:: dependency-injection/_static/request-services.png
 
-Request Services represent the services you configure and request as part of your application. Application Services are limited to those things that are available on application startup. Anything that is scoped is only available as part of Request Services, not Application Services. When your objects specify dependencies, these are satisfied by the types found in ``RequestServices``, not ``ApplicationServices``.
+Request Services represent the services you configure and request as part of your application. When your objects specify dependencies, these are satisfied by the types found in ``RequestServices``, not ``ApplicationServices``.
 
 Generally, you shouldn't use these properties directly, preferring instead to request the types your classes you require via your class's constructor, and letting the framework inject these dependencies. This yields classes that are easier to test (see :doc:`/testing/index`) and are more loosely coupled.
 
-.. note:: The important things to remember are that your application will almost always use ``RequestServices``, and in any case you shouldn't access these properties directly. Instead, request the services you need via your class's constructor. 
+.. warning:: Prefer requesting dependencies as constructor parameters to accessing the ``RequestServices`` collection. 
```

Comment:
Ok. I've gotten pretty strong guidance from the team about avoiding using RequestServices so was trying to be a bit more firm about it.